app/submitter/converter.py

```diff
@@ -95,15 +95,15 @@ def prepare_responses(user, metadata_store, questionnaire, responses):
         paradata = {}
         submitted_at = datetime.now(settings.EUROPE_LONDON)
 
-        response = {SubmitterConstants.TYPE_KEY: SubmitterConstants.TYPE,
-                    SubmitterConstants.VERSION_KEY: SubmitterConstants.VERSION,
-                    SubmitterConstants.ORIGIN_KEY: SubmitterConstants.ORIGIN,
-                    SubmitterConstants.SURVEY_ID_KEY: survey_id,
-                    SubmitterConstants.SUBMITTED_AT_KEY: submitted_at.strftime(settings.DATETIME_FORMAT),
-                    SubmitterConstants.COLLECTION_KEY: collection,
-                    SubmitterConstants.METADATA_KEY: metadata,
-                    SubmitterConstants.PARADATA_KEY: paradata,
-                    SubmitterConstants.DATA_KEY: data}
-
-        logging.debug("Converted response ready for submission %s", json.dumps(response))
-        return response, submitted_at
+        payload = {SubmitterConstants.TYPE_KEY: SubmitterConstants.TYPE,
+                   SubmitterConstants.VERSION_KEY: SubmitterConstants.VERSION,
+                   SubmitterConstants.ORIGIN_KEY: SubmitterConstants.ORIGIN,
+                   SubmitterConstants.SURVEY_ID_KEY: survey_id,
+                   SubmitterConstants.SUBMITTED_AT_KEY: submitted_at.strftime(settings.DATETIME_FORMAT),
+                   SubmitterConstants.COLLECTION_KEY: collection,
+                   SubmitterConstants.METADATA_KEY: metadata,
+                   SubmitterConstants.PARADATA_KEY: paradata,
+                   SubmitterConstants.DATA_KEY: data}
+
+        logging.debug("Converted answer ready for submission %s", json.dumps(payload))
+        return answer, submitted_at
```

Comment:
This isn't going to work mate, this should be payload not answer. I can't see how this even works?